tripzone/src/containers/right-container/RightContainer.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import styles from "./RightContainer.module.scss"
+import TripForm from '../trip-form/TripForm';
+import AvailableFlights from '../available-flights/AvailableFlights';
+import BookFlight from '../book-flight/BookFlight';
+import { isEmpty } from "lodash"
+import getAvailableFlights from '../../services/getAvailableFlights';
+
+
+const RightContainer = (props) => {
```

Comment:
could you let me know if this naming is appropriate one and also are we using props inside this